tests/dashboards/api_tests.py

```diff
@@ -78,6 +82,32 @@ def insert_dashboard(
         db.session.commit()
         return dashboard
 
+    @pytest.fixture()
+    def create_dashboards(self):
+        with self.create_app().app_context():
+            dashboards = []
+            admin = self.get_user("admin")
+            for cx in range(DASHBOARDS_FIXTURE_COUNT - 1):
+                dashboards.append(
+                    self.insert_dashboard(f"title{cx}", f"slug{cx}", [admin.id])
+                )
+            fav_dashboards = []
+            for cx in range(round(DASHBOARDS_FIXTURE_COUNT / 2)):
+                fav_star = FavStar(
+                    user_id=admin.id, class_name="Dashboard", obj_id=dashboards[cx].id
+                )
+                db.session.add(fav_star)
+                db.session.commit()
+                fav_dashboards.append(fav_star)
+            yield dashboards
+
+            # rollback changes
+            for dashboard in dashboards:
```

Comment:
is that rollback will also remove from dashboard_user table and dashboard_slices ?

Reply:
Yes it cleans up after `dashboard_user` and no dashboard_slices are added so nothing to do here